CFC_WebApp/clients/socialmedia/result_template.tpl

```diff
@@ -0,0 +1,98 @@
+<html>
+<head>
+<title>Facebook Login JavaScript Example</title>
+<meta charset="UTF-8">
+</head>
+<body>
+<script>
+  // This is called with the results from from FB.getLoginStatus().
+  function statusChangeCallback(response) {
+    console.log('statusChangeCallback');
+    console.log(response);
+    // The response object is returned with a status field that lets the
+    // app know the current login status of the person.
+    // Full docs on the response object can be found in the documentation
+    // for FB.getLoginStatus().
+    if (response.status === 'connected') {
+      // Logged into your app and Facebook.
+      testAPI(response.authResponse.accessToken);
+    } else if (response.status === 'not_authorized') {
+      // The person is logged into Facebook, but not your app.
+      document.getElementById('status').innerHTML = 'Please log ' +
+        'into this app.';
+    } else {
+      // The person is not logged into Facebook, so we're not sure if
+      // they are logged into this app or not.
+      document.getElementById('status').innerHTML = 'Please log ' +
+        'into Facebook.';
+    }
+  }
+
+  // This function is called when someone finishes with the Login
+  // Button.  See the onlogin handler attached to it in the sample
+  // code below.
+  function checkLoginState() {
+    FB.getLoginStatus(function(response) {
+      statusChangeCallback(response);
+    });
+  }
+
+  window.fbAsyncInit = function() {
+  FB.init({
+    appId      : '456261131199824',
```

Comment:
Oops. Yes. Thanks! I’ll pass it in on as a template variable.
Sid Feygin
sid.feygin@gmail.com



> On Jun 2, 2015, at 4:59 PM, shankari <notifications@github.com> wrote:
> 
> In CFC_WebApp/clients/socialmedia/result_template.tpl <https://github.com/e-mission/e-mission-server/pull/133#discussion_r31583445>:
> 
> > +        'into Facebook.';
> > +    }
> > +  }
> > +
> > +  // This function is called when someone finishes with the Login
> > +  // Button.  See the onlogin handler attached to it in the sample
> > +  // code below.
> > +  function checkLoginState() {
> > +    FB.getLoginStatus(function(response) {
> > +      statusChangeCallback(response);
> > +    });
> > +  }
> > +
> > +  window.fbAsyncInit = function() {
> > +  FB.init({
> > +    appId      : '456261131199824',
> Shouldn't this be part of the keys as well?
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/e-mission/e-mission-server/pull/133/files#r31583445>.
>